client/src/app/TabsProvider.js

```diff
@@ -698,7 +698,7 @@ function replaceVersions(contents) {
 
   const latestCloudVersion = ENGINE_PROFILES.find(
     p => p.executionPlatform === ENGINES.CLOUD
-  ).executionPlatformVersions[0];
+  ).latestStable;
```

Comment:
I propose to apply this new feature consistently, i.e. also for C7 (https://github.com/camunda/camunda-modeler/pull/3286/files#diff-a8ee53f63c47900983b24616f87f9b04023e1b48be19fd3ee726d0138276b811L697):

Reply:
Cf. https://github.com/camunda/camunda-modeler/pull/3286/commits/959d1d693284197568a902a931ef16b680e0a226 and https://github.com/camunda/camunda-modeler/pull/3286/commits/7cc30096f26b843cfeba6cda9a8f36322bc6b5c5.

Reply:
makes sense to me. Should we then also support pre-release Camunda 7 versions? 7.19 has an alpha out already (cf. https://docs.camunda.org/enterprise/download/,  [94dccc4](https://github.com/camunda/camunda-modeler/pull/3286/commits/94dccc4b103bb25cf47118caaf33aa0354414af2))

Reply:
Yes in this case! :)

Reply:
👍 

I'll squash the commits real quick